app/controllers/concerns/filter_parameters.rb

```diff
@@ -22,9 +22,12 @@ def merge_previous_parameters(all_parameters)
   end
 
```

Comment:
The methods below here are all private. I think the private was removed in https://github.com/DFE-Digital/find-teacher-training/commit/28bc26a1b49c1bffaba9d9ff18fda91c59c8bdc4#diff-6b93dd519c02510c57309a985518c560 possibly by mistake. Could we put it back in?